tests/errors/ElabLocation3.hs

```diff
@@ -183,28 +183,6 @@ part1 _ = do
     let allRounds2 = iterate {- (Main.round 8 3 m2) -} id input in
       print $ allRounds2 !! 20
 
-part2 :: () -> IO ()
```

Comment:
Why delete this? (Does it break something?)

Reply:
Verification is expected to fail with {-@ LIQUID "--expect-error-containing=ElabLocation3.hs:174:1-5" @-}.

One peculiarity, however, is that verification fails with a crash, which means that other errors are not produced that would otherwise happen if verification didn't end abruptly.

If the order in which errors are encountered changes, part2 produces a similar crash to the expected crash in part1. But if part2 is encountered first, then part1 doesn't produce the expected error.

By removing part2, I only leave in the file one source of crashes, and the test becomes more reliable.